app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<section>
+
+  <h2 class="text-center mb-3">Public Recipes</h2>
+
+  <%@recipes.each do |recipe|%>
+    <div class="border border-2 border-danger p-2 bg-dark text-white p-3 rounded mb-3">
+      <div class="row d-flex justify-content-around align-items-center">
+        <div class="col-3 text-center">
+          <p class="fw-bold fs-3"><%=recipe.name%></p>
+          <p>by <%= recipe.user.name %></p>
+        </div>
+
+        <div class="col-6">
+          <p>Total Food Item: <span class="badge bg-danger"> <%= recipe.total_food_items %> </span></p> 
+          <p>Total Price: <span class="badge bg-danger"><%= recipe.total_price %> </span></p>
+        </div>
+      </div>
+    </div>
+  <%end%>
+
+</div>
```

Comment:
- [ ] Here the user should be able to click the public recipe which will then lead to  recipe details